Exericse1.1.4

ashpace opened the pull request:
Please approve my branch

ashpace committed:
Exericse1.1.4

GiuseppeMascolo commented on E1_1_4.m:
```diff
@@ -0,0 +1,48 @@
+meu_L=.1;
+sigma2_L=.2;
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate Student sample with above parameters
```
this comment isn't complete you are also generating a Lognormal

GiuseppeMascolo commented on E1_1_4.m:
```diff
@@ -0,0 +1,48 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate Student sample with above parameters
+
+z = -1: 0.01 : 3; 
```
why between -1 and 3?

GiuseppeMascolo commented on E1_1_4.m:
```diff
@@ -0,0 +1,48 @@
+plot(z,pdfZ)
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate plot of samples
+pdX = makedist('tLocationScale', 0, sqrt(0.1), 8);
```
you are creating again a tStudent distribution, why don't use the same?

GiuseppeMascolo commented on E1_1_4.m:
```diff
@@ -0,0 +1,48 @@
+scale = w:w:1;
+
+figure
+plot(ZSample, scale, '.');
```
why are you plotting the Zsample in scale belonging to the interval [1/n,1]??

ashpace replied:
Good question

GiuseppeMascolo reviewed (changes requested):
You need to ad the empirical distribution and empirical quantile.
Great job!

ashpace reviewed (commented):
The comments are fair and will be addressed shortly

ashpace committed:
Draft 2 - Ready for pull

Readme created for the folder.
Exercise is now compared by model answer.
Graphs are labled

ashpace commented:
@GiuseppeMascolo your concerns are addressed. Please review for approval. X Ash
